qutebrowser/browser/qutescheme.py

```diff
@@ -201,6 +203,27 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('tabs')
+def qute_tabs(_url):
+    """Handler for qute://tabs. Display information about all open tabs."""
+    tabs = collections.defaultdict(list)
+    for win_id, window in objreg.window_registry.items():
+        if sip.isdeleted(window):
+            continue
+        tabbed_browser = objreg.get('tabbed-browser',
+                                    scope='window',
+                                    window=win_id)
+        for tab in tabbed_browser.widgets():
+            if tab.url() != QUrl("qute://tabs/"):
```

Comment:
Hmm, somehow this doesn't always seem to work correctly, especially when refreshing the page.

I tried `tab.url().matches(QUrl("qute://tabs/"), QUrl.StripTrailingSlashes)` but that also doesn't seem to match correctly? 😕 

I suggest you do `if tab.url() not in [QUrl("qute://tabs/"), QUrl("qute://tabs")]` or so instead...

However, I think there's a different issue: When refreshing the page, I see `qute://tabs` sometimes, and I'm wondering whether this is because the page is loading (and the URL is getting set) while we're doing this here. If the comparison change above doesn't help, can you try whether using `tab.url(requested=True)` does?